packages/shared/src/components/squads/moderation/SquadModerationItem.tsx

```diff
@@ -0,0 +1,110 @@
+import React, { MouseEvent, ReactElement } from 'react';
+import {
+  Typography,
+  TypographyTag,
+  TypographyType,
+} from '../../typography/Typography';
+import { CardImage } from '../../cards/common/Card';
+import PostTags from '../../cards/common/PostTags';
+import {
+  SourcePostModeration,
+  SourcePostModerationStatus,
+} from '../../../graphql/squads';
+import { SquadModerationActions } from './SquadModerationActions';
+import { ProfileImageSize, ProfilePicture } from '../../ProfilePicture';
+import PostMetadata from '../../cards/common/PostMetadata';
+import { AlertPointerMessage } from '../../alert/common';
+import { SourceMemberRole, Squad } from '../../../graphql/sources';
+import { Button, ButtonSize, ButtonVariant } from '../../buttons/Button';
+import { capitalize } from '../../../lib/strings';
+import OptionsButton from '../../buttons/OptionsButton';
+import { TimerIcon } from '../../icons';
+import { WarningIcon } from '../../icons/Warning';
```

Comment:
If you export warning from icons we can do 1 barrel import here.

Reply:
When do barrel imports work for us? I remember some discussion to avoid it as much as possible, is it not the case with the icons?

Reply:
With Icons it's normally safe as they don't export other functions that can cause infinite cycle.
For now let's keep using them for icons.

Reply:
Yeah, that makes sense. Thank you!